packages/health-check/src/verboseChecks.js

```diff
@@ -69,13 +70,42 @@ function dappExampleCheck(done) {
   skylinkVerification(done, linkInfo);
 }
 
+const developMomentumBodyHash = "53b44a9d3cfa9b3d66ce5c29976f4383725d3652";
+const developMomentumMetadata = JSON.parse(fs.readFileSync("./fixtures/developMomentumMetadata.json"));
```

Comment:
```suggestion
const developMomentumMetadata = require("./fixtures/developMomentumMetadata.json");
```

Reply:
well isn't that fancy. I assumed it was going to be something neat like this but I didn't know what to google haha.